src/stories/5-AbortMessage.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
```

Comment:
Great job with the stories. 
Lets start colocating stories along side their respective components.